RELEASE-NOTES.txt

```diff
@@ -8,7 +8,6 @@
 21.0
 -----
 - [***] Merchants can now view and edit custom fields of their products and orders from the app [https://github.com/woocommerce/woocommerce-ios/issues/13493]
-- [*] Payments onboarding: for two onboarding errors that show a wp-admin Payments setup URL in a webview ("Payments plugin not setup" and "Stripe overdue" errors that can occur to both WooPayments and Stripe Extension), the webview is now pre-authenticated for merchants with site credentials login [https://github.com/woocommerce/woocommerce-ios/pull/14252]
```

Comment:
Good point, I feel like the release notes read well by just taking out the two sentences `Also, we've smoothed out the payments onboarding process. It's now easier than ever to resolve errors setting up WooPayments or Stripe.` for the English version. @rachelmcr do you think it's okay for me to update the [user-facing release notes](https://github.com/woocommerce/woocommerce-ios/blob/3087192c9da65e987e2688d45a3c0ba104a8192a/WooCommerce/Resources/release_notes.txt) in this PR? I'm not sure if it might interrupt the translation and submission process. I plan to merge the PR in an hour.

Reply:
> do you think it's okay for me to update the [user-facing release notes](https://github.com/woocommerce/woocommerce-ios/blob/3087192c9da65e987e2688d45a3c0ba104a8192a/WooCommerce/Resources/release_notes.txt) in this PR? I'm not sure if it might interrupt the translation and submission process.

I'm fairly sure changing it at this point will invalidate any existing translations, so we'd have to resubmit the text for translation (which might not get done in time). I think I'd err on the side of leaving the release notes as they are. It probably won't do any harm to have that mention in the release notes, whereas not having the release notes translated would be more disruptive.

Reply:
Makes sense! I just merged the PR, thanks in advance @rachelmcr for making the beta build 🙏